### PR TITLE
Add ax_enabled for 5Ghz band

### PR DESCRIPTION
**Pull request by hannut**

### CODE REVIEW

**Comment by hannut** [comment]
I fixed the indentations with ALEFix. Seems that it also changed quotes

**Comment by hannut** [comment]
First patch adds the setting. Second one is for tests to go trough. I think it's ready

**Review by kbreit — CHANGES_REQUESTED** [review]

File: plugins/modules/meraki_mr_rf_profile.py [hunk 1]
```diff
@@ -449,6 +461,8 @@ def construct_payload(meraki):
             payload['fiveGhzSettings']['rxsop'] = meraki.params['five_ghz_settings']['rxsop']
         if meraki.params['five_ghz_settings']['channel_width'] is not None:
             payload['fiveGhzSettings']['channelWidth'] = meraki.params['five_ghz_settings']['channel_width']
+        if meraki.params['five_ghz_settings']['ax_enabled'] is not None:
```
[the remark this answers is not in the document]
Comment (hannut) [reply plugins/modules/meraki_mr_rf_profile.py:464]: There is ax setting on both bands on the dashboard. However I tried flipping the 5ghz setting on and off on the dashboard and then tried to change it with ansible and it actually doesn't do anything. Maybe the public api is behind what can be done on the meraki dashboard

Kuvakaappaus - 2023-01-18 07-23-05

**Comment by kbreit** [comment]
@hannut Is there a reason you're closing this PR? The 2.4ghz is doable. It's just the 5ghz setting that is invalid in the API.

**Pull request closed without merging**